### PR TITLE
[KEYCLOAK-7826] - My resource pages does not support resources without scopes

### PR DESCRIPTION
**Pull request by pedroigor**

### CODE REVIEW

**Review by mhajas — APPROVED** [review]

**Comment by stianst** [comment]
@keycloak-ci-bot test

**Review by stianst — APPROVED** [review]
Approving without review, assuming @mhajas has reviewed

**Comment by stianst** [comment]
This PR failed on Travis and was not completed on Jenkins CI before merging. Please wait for green light before merging.

**Pull request merged**
